### DISCUSSION TITLE
Python simple dice

### DISCUSSION THREAD

**Discussion by Ramutis462** [community/community#21516] [Other Feature Feedback, Questions, & Ideas]
<p>Why this simple python code always prints “incorrect” ?</p>
<p>import random</p>
<p>random_number = random.randint(1, 6)<br>
guess = 0</p>
<p>while True:<br>
guess = raw_input("Enter random number between 1-6: ")<br>
if guess == random_number:<br>
print(“correct”)</p>
<p>else:<br>
print(“incorrect”)</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Read up on raw_input(). It returns a string.</p>

**Comment by Ramutis462** [comment]
<p>Is this how to fix it?</p>
<p>int(input())</p>

**Comment by Ramutis462** [comment]
<p>The code:</p>
<p>import random</p>
<p>random_number = random.randint(1, 6)</p>
<p>while True:<br>
guess = int(raw_input("Enter random number between 1-6: "))<br>
if guess == random_number:<br>
print(“correct”)</p>
<p>else:<br>
print(“incorrect”)</p>
<p>The output:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/86ab9ee4b4689f8578119f5f71dcfa788d5e96b8.png" data-download-href="/uploads/short-url/jdlFX1xOQl79xoznBT1bjNulgFO.png?dl=1" title="Output.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/59694550/181065088-7cfd50a7-365d-43e2-834d-dee0194bfc91.png" alt="Output.PNG" data-base62-sha1="jdlFX1xOQl79xoznBT1bjNulgFO" width="466" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/86ab9ee4b4689f8578119f5f71dcfa788d5e96b8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Output.PNG</span><span class="informations">957×1025 6.13 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Ramutis462** [comment]
<p>Found solution, thanks for help</p>
<p>import random</p>
<p>while True:<br>
random_number = random.randint(1, 6)<br>
guess = int(raw_input("Enter random number between 1-6: "))<br>
if guess == random_number:<br>
print(“correct”)</p>
<p>else:<br>
print(“incorrect”)</p>
